tools/crm_mon_print.c

```diff
@@ -206,9 +206,8 @@ print_resources(pcmk__output_t *out, pe_working_set_t *data_set,
          */
         } else if (!partially_active && is_not_set(mon_ops, mon_op_inactive_resources)) {
             continue;
-        }
 
-        if (!pe__rsc_running_on_any_node_in_list(rsc, only_show)) {
+        } else if (is_active && !pe__rsc_running_on_any_node_in_list(rsc, only_show)) {
```

Comment:
I think this should be is_partially_active, am I missing something?

Reply:
Nope, not missing anything.  I keep forgetting that partially_active is true for fully active, too.  Pushed a new patch set with just that change.